test/nools/target-emitter.spec.js

```diff
@@ -149,6 +149,27 @@ describe('target emitter', () => {
         ]);
       });
 
+      it('should default to "now" if target date is a function returning null', () => {
```

Comment:
Please add a test for the case when date is assigned null,  `target.date = null;`

Reply:
I have tried to add it and got an error, then tried to upload a setting config with a `target.date = null`, cht-conf validations step rejected if, that's why I did not add that test. 
<img width="1265" alt="image" src="https://user-images.githubusercontent.com/6564732/204819404-3b6ecfd3-176f-494f-a3ce-1e3760e69b92.png">

Reply:
Do you get the same if you assign explicitly `undefined`?

Reply:
`undefined` passes the validation...

Reply:
Okay, but I think that case is already covered below, right?

Reply:
I think so, it's undefined if no value is set, but I can add an unit test where I explicitly set the date to undefined, if you find it useful.

Reply:
No, I think like this is okay